static/redfish/v1/JsonSchemas/OemPowerSupplyMetric/index.json

```diff
@@ -0,0 +1,109 @@
+            },
+            "type": "object"
+        },
+        "IBM": {
+            "additionalProperties": true,
+            "description": "IBM Oem propperties for Power Supply Metrics.",
+            "patternProperties": {
+                "^([a-zA-Z_][a-zA-Z0-9_]*)?@(odata|Redfish|Message)\\.[a-zA-Z_][a-zA-Z0-9_]*$": {
+                    "description": "This property shall specify a valid odata or Redfish property.",
+                    "type": [
+                        "array",
+                        "boolean",
+                        "integer",
+                        "number",
+                        "null",
+                        "object",
+                        "string"
+                    ]
+                }
+            },
+            "properties": {
+                "InputPowerHistoryItem": {
+                    "$ref": "#/definitions/InputPowerHistoryItem",
+                    "description": "Contains power supply input power history.",
+                    "longDescription": "Contains power supply input power history, timestamped average and maximum input power over 30 second intervals.",
+                    "readonly": false,
+                    "versionAdded": "v1_0_0"
+                }
+            }
+        },
+        "InputPowerHistoryItem": {
+            "additionalProperties": false,
+            "description": "Contains power supply input power history.",
+            "longDescription": "Contains power supply input power history, timestamped average and maximum input power over 30 second intervals.",
+            "patternProperties": {
+                "^([a-zA-Z_][a-zA-Z0-9_]*)?@(odata|Redfish|Message)\\.[a-zA-Z_][a-zA-Z0-9_]*$": {
+                    "description": "This property shall specify a valid odata or Redfish property.",
+                    "type": [
+                        "array",
+                        "boolean",
+                        "integer",
+                        "number",
+                        "null",
+                        "object",
+                        "string"
+                    ]
+                }
+            },
+            "properties": {
+                "Date": {
+                    "description": "Timestamp for when records were retrieved",
+                    "format": "date-time",
+                    "longDescription": "This property shall contain an array of objects representing the timestamp for when a record was retrieved from the power supply.",
+                    "readonly": true,
+                    "type": "string"
+                },
+                "Average": {
+                    "description": "The average power supply input power values.",
+                    "longDescription": "This property shall contain an array of objects that represent the average input power the power supply has calculated, each over a 30 second interval.",
+                    "readonly": true,
+                    "type": "array"
```

Comment:
I am surprised this an array.. I would have thought this is an integer or number? Same with Max

Reply:
Um... I made this an array, and followed examples of others using an array, based on a conversation with Gunnar. :-)

Reply:
Oh, I thought they would be this 
```
 "PowerSupplyInputData": [
 {"2022-04-21T16:41:52+00:00", 10, 10},
 {"2022-04-21T16:41:22+00:00", 11, 11}
]

```

Vs

```
  "Average": [
    10,
    10
  ],
  "Date": [
    "2022-04-21T16:41:52+00:00",
    "2022-04-21T16:41:22+00:00"
  ],
  "Id": "Metrics",
  "Max": [
    10,
    10
  ],
 ```

Reply:
I am okay with how you did it

Reply:
The format Shawn mentioned does seem better, but now I am struggling to understand what the schema changes would look like:
```
  "Oem": {
    "IBM": {
      "InputPowerHistory": [
        {
          "Average": 205,
          "Date": "2022-04-21T19:10:35+00:00",
          "Max": 205
        },
        {
          "Average": 144,
          "Date": "2022-04-21T19:10:05+00:00",
          "Max": 144
        }
      ]
    }
  }
```